app/pb/community.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+package org.couchers.community;
+import "google/protobuf/timestamp.proto";
+
+import "pb/nullables.proto";
+
+message Point{
+  float lat = 1;
+  float lng = 2;
```

Comment:
These need to be doubles

Reply:
Do we need more than 4 meters resolution?

Reply:
Yes. There's also no reason to use floats instead